CSharp/BatchExplorer/Views/MainView.xaml

```diff
@@ -373,6 +373,43 @@
             </StackPanel>
                 
         </ControlTemplate>
+        
+        <ControlTemplate x:Key="CertificateCommands">
+            <StackPanel
+                Orientation="Vertical">
+                <StackPanel
+                    Orientation="Horizontal">
+                    <Button
+                        Margin="4"
+                        Width="48"
+                        Command="{Binding RefreshItem}"
```

Comment:
RefreshItem is existing general-purpose command - after discussion with Pablo, we are okay to ignore this comment